### ISSUE TITLE
Preparation for team review of WCAG 2.2 content updates

### ISSUE THREAD

**Issue by calvin-lau-sig7** [alphagov/govuk-design-system#3338]
## What

Organise WCAG 2.2 edits for a 2i review, distributed amongst the team. Prepare needed documents.

Document to organise everyone should be new tab in the [Pages for WCAG 2.2 spreadsheet](https://docs.google.com/spreadsheets/d/1cfuUiYQMN4XPkYTW9jmxdpFvy3ttaaPeFNRso-NC2yE/edit?usp=sharing)

Should include
- list of pages to look at
- criteria to check, along with basic guidance – probably accuracy and structure/presentation ([based on 2i guidance](https://docs.google.com/document/d/1xMcouXZxeuxsjtD0d-ArFVqlf1z8CaOQqNzsmKn0Wos/edit?usp=sharing))
- assignments of pages for team members to look at, or space to capture when pages have been looked at
- link to content updates doc, for reference if needed

## Why

This should act as a 'sense check' that all updates look good and make sense on our website.

## Who needs to review

To review the 'organise docs' – Calvin, Charlotte, Kelly

## Down when
- [x] prepare list of pages that need to be checked
- [x] decide and share checklist of things to look for
- [x] recruit reviewers
- [x] assign areas or pages for reviewers to look at

**Comment by kellylee-gds** [comment]
Dev / designer pair to answer outstanding questions. If unsure, check with Anika (at work until 21 Dec).

Anyone can do the 2i.

**Comment by kellylee-gds** [comment]
Closing this issue as team review has begun. Deadline of 22 Dec set.
